appinfo/Migrations/Version20170913113840.php

```diff
@@ -0,0 +1,48 @@
+<?php
+/**
+ * @author Sujith Haridasan <sharidasan@owncloud.com>
+ *
+ * @copyright Copyright (c) 2017, ownCloud GmbH
+ * @license AGPL-3.0
+ *
+ * This code is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Affero General Public License, version 3,
+ * as published by the Free Software Foundation.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU Affero General Public License for more details.
+ *
+ * You should have received a copy of the GNU Affero General Public License, version 3,
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>
+ *
+ */
+
+namespace OCA\Encryption\Migrations;
+
+use OCP\Migration\ISimpleMigration;
+use OCP\Migration\IOutput;
+
+class Version20170913113840 implements ISimpleMigration {
+
+	/**
+	 * @param IOutput $out
+	*/
+	public function run(IOutput $out) {
+		$installedVersion = \OC::$server->getConfig()->getSystemValue('version', '0.0.0');
+
+		if ($installedVersion !== '0.0.0') {
+			$maxVersion = explode('.', $installedVersion)[0];
```

Comment:
there's a more PHP-y way to do this, see https://github.com/owncloud/core/blob/master/lib/private/Repair/Apps.php#L99